README.md

```diff
@@ -1 +1,148 @@
-# final-capstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# We Fly Digital
+
+> We Fly Digital is a responsive app that lists flight reservations. A user can selects a specific airline, see the details page with its full description, and can click the "Reserve" button to reserve it. The user can also access the "Reserve" page from the navigation panel, add or delete a reservation or airline.
+
+
+## API Documentation
+Check out the [API documentation](https://flight-digital.herokuapp.com/api-docs/index.html)
+
+## Project Documentation 📄
+
+- [x] Here is the deployed of back end [Heroku](https://flight-digital.herokuapp.com/)👈
+- [x] Here is the deployed of front end [Heroku](https://we-fly-digital.herokuapp.com/)👈
+
+## Kanban board 
+
+- [x] [Link to the initial state of our Kanban board](https://user-images.githubusercontent.com/86069740/194050526-743a478f-d217-40b3-ae1d-11708bd5e355.png)
+- [x] [Link to the final state of our Kanban board](https://github.com/users/ibr5500/projects/6)
+
+### ERD diagram
+
+![Screenshot from 2022-10-05 10-32-43](https://user-images.githubusercontent.com/86069740/194005586-9b6bced6-eca9-41ec-b2f4-7ece0cd23615.png)
+
+### Front-End Repo 
+
+- [x] [We Fly Digital](https://github.com/ibr5500/flight_reservation-frontend)
+
+## Built With 🛠️
+
+- [x] Programming Language: [Ruby](https://www.ruby-lang.org/en/)
+- [x] Framework: [Ruby on Rails](https://rubyonrails.org/)
+- [x] Linter: [Rubocop](https://rubocop.org/)
+- [x] Code Editor: [VS Code](https://code.visualstudio.com/)
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Ruby.
+
+You can check if Git is installed by running the following command in the terminal.
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+```
+$ ruby --version && irb
+```
+
+To install rails, in the terminal kindly run this command
+```
+$ gem install rails
+```
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+## installation
+
+To get a local copy up and running follow these simple example steps.
+
+- clone the repository by running: git@github.com:ibr5500/flight_reservation-backend.git
+
+- setup rails app
+``` rails . my_api --api --database=postgresql```
+- navigate to the folder
+  ``` cd <DIRECTORY> ```
+- Install packages
+  ``` npm i ```
+- Run the app
+```sh
+# Load the schema
+rails db:schema:load
+# Create the database
+rake db:create
+# Run database Migration
+rails db:migrate
+```
+### Usage
```

Comment:
Some of these instrucitons are not necessarily tellign me how to run this backed app.
From cloning to running app with rails server,  I should be able to follow instructions and run application.

Where does `npm i` come in with this app?